components/ContentTemplates/ImagesTemplate.tsx

```diff
@@ -1,8 +1,408 @@
+						<area
+							shape="circle"
+							coords="190,130,70"
+							href="https://en.wikipedia.org/wiki/Doughnut"
+							target="_blank"
+							alt="Doughnut"
+						/>
+						<area
+							shape="circle"
+							coords="380,250,100"
+							href="https://en.wikipedia.org/wiki/Coffee"
+							target="_blank"
+							alt="Coffee"
+						/>
+					</map>
+					<Image
+						useMap="#primary"
+						src="/images/imagesTemplate/donut-coffee.png"
+						width={600}
+						height={400}
+						alt="Illustration of a pink doughnut with sprinkles next to a cup of black coffee"
+						layout="intrinsic"
+					/>
+					<CodeBlock
+						codeSnippet={`<map name="primary">
+	<area
+		shape="circle"
+		coords="190,130,70"
+		href="https://en.wikipedia.org/wiki/Doughnut"
+		alt="Doughnut"
+	/>
+	<area
+		shape="circle"
+		coords="380,250,100"
+		href="https://en.wikipedia.org/wiki/Coffee"
+		alt="Coffee"
+	/>
+</map>
+<img
+	usemap="#primary"
+	src="donut-coffee.png"
+	alt="Illustration of a pink doughnut with sprinkles next to a cup of black coffee" 
+/>`}
+					/>
+				</TemplateSection>
+				{/* TODO: <TemplateSection sectionName="checklist"  title="Images Checklist">
+					
+					<p>Add some points here</p>
+				</TemplateSection> */}
+				<TemplateSection sectionName="WCAGCriteria" title="WCAG Criteria">
+					<a
+						href="https://www.w3.org/TR/WCAG21/#non-text-content"
+						className="blockLink">
+						1.1.1 Non-text content
+					</a>
+					<a
+						href="https://www.w3.org/TR/WCAG21/#use-of-color"
+						className="blockLink">
+						1.4.1 Use of Color
+					</a>
+					<a
+						href="https://www.w3.org/TR/WCAG21/#images-of-text"
+						className="blockLink">
+						1.4.5 Images of Text
+					</a>
+					<a
+						href="https://www.w3.org/TR/WCAG21/#images-of-text-no-exception"
+						className="blockLink">
+						1.4.9 Images of Text (No Exception)
+					</a>
+				</TemplateSection>
+				<TemplateSection sectionName="otherResources" title="Other Resources">
+					<a
+						href="https://www.w3.org/WAI/tutorials/images/"
+						className="blockLink">
+						W3C Images Tutorial
+					</a>
+					<a
+						href="https://www.w3.org/WAI/tutorials/images/decision-tree/"
+						className="blockLink">
+						W3C An alt decision tree
+					</a>
+					<a
+						href="https://jakearchibald.com/2021/great-alt-text/"
+						className="blockLink">
+						Writing great alt text: Emotion matters
+					</a>
+					<a
+						href="https://tink.uk/thoughts-on-skin-tone-and-text-descriptions.md-notes-on-synthetic-speech/"
+						className="blockLink">
+						Thoughts on skin tone and text descriptions
+					</a>
+					<a
+						href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/map"
+						className="blockLink">
+						MDN: The Image Map element
+					</a>
+				</TemplateSection>
+			</article>
+			<p className="pageLastUpdated">
+				<strong>Page last updated: </strong>26th November 2022
+			</p>
+		</>
+	)
 }
 
 export default ImagesTemplate
```

Comment:
To stay consistent with the named exports, you can remove this and export the component above.